src/main/java/org/jvnet/hudson/test/BuildWatcher.java

```diff
@@ -90,7 +89,7 @@
                 return;
             }
             RunningBuild build = new RunningBuild(r);
-            RunningBuild orig = builds.put(r.getLogFile(), build);
+            RunningBuild orig = builds.put(r.getRootDir(), build);
```

Comment:
If you are referring to logging from non-`Run`s, that is true, but anyway `BuildWatcher` does not support them currently either.